src/components/OfflineWithFeedback.js

```diff
@@ -74,10 +74,7 @@ function applyStrikeThrough(children) {
 
 const OfflineWithFeedback = (props) => {
     const hasErrors = !_.isEmpty(props.errors);
-    const isOfflinePendingAction = props.network.isOffline && props.pendingAction;
-    const isUpdateOrDeleteError = hasErrors && (props.pendingAction === 'delete' || props.pendingAction === 'update');
-    const isAddError = hasErrors && props.pendingAction === 'add';
-    const needsOpacity = (isOfflinePendingAction && !isUpdateOrDeleteError) || isAddError;
+    const needsOpacity = props.network.isOffline && props.pendingAction === 'add';
```

Comment:
@iwiznia, can you double check this code. As per the [pattern B](https://docs.google.com/document/d/1Ija9CHiNBH1HVnnITA-XuohF31BoiseY18a8_BxFtjc/edit#heading=h.mnjguil21u0a) doc pendingAction 'add' should show up with opacity when offline. So I simplified the logic here.

The previous logic here shows that if pendingAction is delete or update with errors then don't show opacity. The updated logic is pretty much the same where nothing happens wrt to opacity for delete or update, but will show opacity for add if its offline if its an error or not.

Let me know if this is alright or if I'm missing something.

Reply:
So, something was broken? If not, I would say leave it as it was.

> The previous logic here shows that if pendingAction is delete or update with errors then don't show opacity.

This is correct, because if update or delete failed, it means the item does exist in the DB, so no opacity is needed.